### DISCUSSION TITLE
Court Interpreter prototype

### DISCUSSION THREAD

**Discussion by legaltextai** [freelawproject/courtlistener#3224] [Show and tell]
[Video](https://www.loom.com/share/ab3cbc8ffebf48968266a708be390a1d?sid=9a970d41-e0fc-4838-93c1-eb2c07640408). Feedback is very welcome.

**Comment by ghost** [comment]
@legaltextai  I am working on a project to search through and analyze public information and I would love to see how you did that

**Reply by legaltextai** [reply]
sure, feel free to email admin at legaltext ai  
pls share what you are building and what problems you are experiencing

**Reply by ghost** [reply]
I'm working on [true-public-access/Seagull](https://github.com/true-public-access/Seagull) don't have any experience with AI and what you've shown is one of the core functionalities I am tying to build.

**Comment by jumpingtom1** [comment]
@legaltextai , how do you see this service as differentiated from keyword searching that currently exists?

**Reply by ghost** [reply]
I know there are keyword searches of the different databases I know Edgar has one and Recap has one but this is distinct because not only access to Edgar and Recap but also other sources like FOI requests and information they publicly provide on their websites and social media accounts. Beyond that, it also helps people understand what they are looking at and its context. You can look at a court filing and say these are the defendants you are the arguments being made but you wouldn't know how this applies to other situations or what exactly certain terms mean. These are some of the challenges people face when using these tools and I'm trying to use AI to bridge the gaps in understanding. Does that explain the differences?

**Comment by legaltextai** [comment]
it was done so long ago, many things have changed :-) i saw it as the first step to getting an answer to your question (based on facts). 
stay tuned.
